### PR TITLE
Format files with Prettier

### PR DESCRIPTION
**Pull request by queengooborg**
This PR formats two files in the repository with Prettier.

### CODE REVIEW

**Comment by hamirmahal** [comment]
Do you think it'd be useful to have a pre-commit formatting check, and/or a GitHub Action to check for formatting errors?

**Comment by queengooborg** [comment]
We actually already have the pre-commit hook!  A GitHub Action is also supposed to run, but it doesn't look like it's configured properly at the moment -- I'll fix that up.

File: live-examples/css-examples/pseudo-class/defined.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <div class="all-elements">
-    <p>This is an ordinary <b>&;lt;p&gt;</b> element.</p>
+  <p>This is an ordinary <b>&lt;p&gt;</b> element.</p>
```
Comment (bsmth) [inline live-examples/css-examples/pseudo-class/defined.html:2]: I'm confused by the original here, I guess it was just a typo.

**Review by bsmth — APPROVED** [review]
Looks good, thanks 👍🏻

**Pull request merged**